pkg/acsengine/defaults.go

```diff
@@ -313,6 +313,24 @@ var (
 			},
 		},
 	}
+
+	// DefaultContainerMonitoringAddonsConfig is the default container monitoring Kubernetes addon Config
+	DefaultContainerMonitoringAddonsConfig = api.KubernetesAddon{
+		Name:    ContainerMonitoringAddonName,
+		Enabled: helpers.PointerToBool(api.DefaultContainerMonitoringAddonEnabled),
+		Config: map[string]string{
+			"omsAgentVersion":       "1.6.0-42",
+			"dockerProviderVersion": "2.0.0-2",
+			"workspaceGuid":         "c5905df1-b3b9-42b9-acf0-14a4c4ef028c",
+			"workspaceKey":          "b4xMPEns/5Oo61hQbRLJQoPhbmAHdBn2eDjkCbRzgEISiKi9m1CR+093sczO9E8iA5w1EyGVCVkwsnuWt1MD9w==",
+		},
+		Containers: []api.KubernetesContainerSpec{
+			{
+				Name:  "omsagent",
+				Image: "dockerio.azureedge.net/microsoft/oms:ciprod05082018",
```

Comment:
Are CPU/Memory Limits and Requests not set intentionally?

Reply:
Yes. This is the yaml file we will be using with AKS cluster creation. We intend to use the same yaml so that we are close to testing out the real scenario.